ovos_config/mycroft.conf

```diff
@@ -611,7 +611,12 @@
     },
     "ovos-tts-plugin-mimic3-server": {
         "voice": "en_UK/apope_low"
-    }
+    },
+    // persistent cache
+    "persist_cache": false,
+    "preloaded_cache": "",
+    "min_free_percent": 75,
+    "persist_thresh": 1
```

Comment:
this should be per plugin

Reply:
Ohhhh, it working at the top level as well.

I'll fix that.